src/components/ec-input-field/ec-input-field.vue

```diff
@@ -44,9 +44,23 @@
       v-model="inputModel"
       :autocomplete="autocomplete"
     >
+    <div
+      v-if="leftIcon"
+      class="ec-input-field__icon-wrapper tw-left-0"
+      :class="{ 'ec-input-field__icon-wrapper--is-disabled': isDisabled }"
+      data-test="ec-input-field__left-icon-wrapper"
+    >
+      <ec-icon
+        class="ec-input-field__icon tw-fill-gray-5"
+        data-test="ec-input-field__left-icon"
+        :name="leftIcon"
+        :size="leftIconSize"
+        :type="leftIconType"
+      />
+    </div>
     <div
       v-if="isLoading || icon"
-      class="ec-input-field__icon-wrapper"
+      class="ec-input-field__icon-wrapper tw-right-0"
```

Comment:
We don't use TW classes directly in the HTML of reusable components unless it's a deviation from the provided design. We should create separate classes like `ec-input-field__left-icon-wrapper` or it could be modifier `ec-input-field__icon-wrapper--right`. Same for `ec-input-field__icon tw-fill-gray-5` and `ec-input-field__icon-wrapper tw-left-0`.